lms/static/coffee/src/instructor_dashboard/student_admin.coffee

```diff
@@ -223,6 +224,30 @@ class @StudentAdmin
           full_error_message = interpolate_text(error_message, {student_id: unique_student_identifier})
           @$request_response_error_ee.text full_error_message
 
+  # Mark a student to skip entrance exam
```

Comment:
@andy-armstrong has posted a reference link in a comment on #6668:  https://github.com/edx/edx-platform/pull/6668#discussion_r24912664